js/webpack.config.js

```diff
@@ -131,6 +132,16 @@ module.exports = {
         ]
       }),
       new HappyPack({
+        id: 'less',
```

Comment:
So it's not used by any file, but it was there previously, so not sure...
Should we get completely rid of LESS ? (ie. in Webpack and `package.json`)